styles/portfolio.css

```diff
@@ -0,0 +1,58 @@
+* {
+    /* border: 1px solid black; */
+    color: #2F3C7E;  
+}
+
+::selection {
+    background-color: #2F3C7E;
+    color: #f6edee;
+}
+
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    background-color: #f6edee;
+}
+
+h2 {
+    background-color: #aec3ba;
+}
+
+a:hover {
+    color: #aec3ba;
+}
+
+a {
+    text-decoration: none;
+    font-style: italic;
+}
+
+ul {
+    text-align: right;
+    font-style: normal;
+    font-weight: bold;
+    display: inline;
+}
+
+p {
+    text-indent: 2em;
+}
+
+#python {
+    width: 10%;
```

Comment:
Consider using a width relative to the font size rather than the overall window. It's a little disconcerting to have the images resize as I resize my browser window.

Reply:
Consider reducing the size of the images to be closer to the intended display size to help reduce the amount of data the visitor must download and improve download speeds.